packages/core/src/sass/app/modules/account-signup.scss

```diff
@@ -75,6 +128,7 @@
     &__btn {
         align-self: flex-end;
         margin: 0;
+        margin-top: -0.5rem;
```

Comment:
can we decrease margin button on the section above button instead?

Reply:
Changed this by subtracting this value from the margin-bottom on __residence-field.